### ISSUE TITLE
Tooltips / popovers?

### ISSUE THREAD

**Issue by peteruithoven** [tbfleming/jscut#14]
Another one for after #12. 
There are quite a few configurations that are in need of a bit of explanation, like _Step over_, _Plunge_ etc. It would be very nice to have short explanations in some kind of tooltips. Bootstrap seems to have them build in: 
- [Popovers](http://getbootstrap.com/javascript/#popovers)
- [Tooltips](http://getbootstrap.com/javascript/#tooltips)

I would recommend the popovers, since it's better suited for a bit more text. 
With a proper show delay this shouldn't be annoying for power users. 

Cura also uses tooltips to explain almost all it's options, I think it's quite useful. 

If you guys like it I'd like to help with the texts.

**Comment by tbfleming** [comment]
Right now there's a popover on "px per inch". Please come up with text for the others; it will save me time.

**Comment by peteruithoven** [comment]
Of course, how could I forget that popup. 
Please correct me where necessary (it will be necessary). I won't complete this in one go, I'll come back to it regularly. 

## Label popups

I would like to add the following explanations to the labels. 

### Operations

**px per inch:** Use to compensate for different pixels per inch translations design applications use. 

### Tool

**Diameter:** Diameter of tool. 
**Pass depth:** Depth the tool should travel per pass. Use a smaller pass depth for harder materials and better quality. 
**Step Over:** What percentage the tool should step over (overlap) the previous path. The more you step over the previously milled path the easier it will be for the drill and the better the end result. But it will take longer. [or was this reversed?]
**Rapid:** The speed during moves, from one milling path to another (while being on clearance height). [not suree if this is right]
**Plunge:** The speed during plunging downwards into the material. 
**Cut:** The speed during milling.

### Selected Operation

**Name:** Name used in gcode comments.
**Visible:** Whether this operation is visible. 
**Ramp Plunge:** [coming soon]
**Combine:** [coming soon]
**Operation:** What operation type is applied on this operation.
**Direction:** In what direction the drill should go. 
**Cut Depth:** How deep this operation should cut in total.
**Margin:** Offset the shape of the milled area outwards or inwards. 
**Width:** 

### Material

**Z Origin:** What is considered the 0 Z position.
**Clearance:** How high the tool moves over the material. Increase this when using bigger clamps or screws to fasten the material. 

### Curve To Line Conversion

**Minimum Segments:**
**Minimum Segment Length (mm):**

## Value popups

### Selected Operation

**Direction:** [coming soon] checking: [wiki](https://en.wikipedia.org/wiki/Milling_cutter#Conventional_milling_versus_climb_milling), [Clim milling vs Conventional milling](http://www.cnccookbook.com/CCCNCMillFeedsSpeedsClimbConventional.htm)

### Material

**Z Origin:**

**Comment by peteruithoven** [comment]
Apparently I don't get mailed by Github when an issue is referenced, thanks for implementing this so quickly. 
I'd really like to help with a good _Ramp Plunge_ explanation, but I don't understand it good enough.
